Sources/ArcGISToolkit/Components/Offline/OfflineMapAreaMetadataView.swift

```diff
@@ -65,7 +65,11 @@ struct OfflineMapAreaMetadataView<Metadata: OfflineMapAreaMetadata>: View {
         .navigationBarTitleDisplayMode(.inline)
         .toolbar {
             ToolbarItem(placement: .confirmationAction) {
-                Button("Done") { dismiss() }
+                Button {
+                    dismiss()
+                } label: {
+                    Text.done
+                }
```

Comment:
Since done buttons are common as well, I recommend adding a convenience method of `Button` for them too:

```swift
extension Button<Text> {
    /// Returns a button with the title "Done" and the given action.
    /// - Parameter action: The action to perform when the user triggers the
    /// button.
    /// - Returns: A button.
    static nonisolated func done(action: @escaping @MainActor () -> Void) -> Self {
        return Button(action: action) {
            Text.done
        }
    }
}
```